rs/replicated_state/src/canister_state/execution_state.rs

```diff
@@ -796,6 +796,33 @@ impl TryFrom<pb::WasmMetadata> for WasmMetadata {
     }
 }
 
+/// Keeps track of how a canister is executing such that the `CyclesAccountManager`
+/// can charge cycles accordingly.
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum WasmExecutionMode {
+    Wasm32,
+    Wasm64,
+}
```

Comment:
Update the comment since this is no longer just used in the `CyclesAccountManager`.

Reply:
done, thanks!